doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1475,6 +1475,90 @@ The Opcodes Aggregator client is an optional Bluetooth mesh model that can be en
 	Abort the Opcodes Aggregator Sequence message. This command clears the Opcodes Aggregator Client context.
 
 
+Remote Provisioning Client
+--------------------------
+
+The Remote Provisioning client is an optional Bluetooth mesh model that can be enabled through the :kconfig:option:`CONFIG_BT_MESH_RPR_CLI` configuration option. The Remote Provisioning Client model is used to support the functionality of dispatching a sequence of access layer messages to nodes supporting the Remote Provisioning Server model.
+
+``mesh models rpr scan <timeout in seconds> [<UUID>]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start scanning for unprovisioned devices.
+
+	* ``timeout in seconds``: Scan timeout in seconds. Must be at least 1 second.
+	* ``UUID``: Device UUID to scan for, or NULL to report all devices.
+
+``mesh models rpr scan-ext <timeout in seconds> <UUID> [<AD-type> ... ]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start extended scanning for unprovisioned devices.
+
+	* ``timeout in seconds``: Scan timeout in seconds. Valid values from @ref BT_MESH_RPR_EXT_SCAN_TIME_MIN to @ref BT_MESH_RPR_EXT_SCAN_TIME_MAX.
+	* ``UUID``: Device UUID to start extended scanning for.
+	* ``AD-type``: List of AD types to include in the scan report. Must contain 1 to :kconfig:option:`CONFIG_BT_MESH_RPR_AD_TYPES_MAX` entries.
+
+``mesh models rpr scan-srv [<AD-type> ... ]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Start extended scanning for the remote server.
+
+	* ``AD-type``: List of AD types to include in the scan report. Must contain 1 to :kconfig:option:`CONFIG_BT_MESH_RPR_AD_TYPES_MAX` entries.
+
+``mesh models rpr scan-caps``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Get scanning capabilities of Remote Provisioning Server.
+
+``mesh models rpr scan-get``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Get current scanning state of Remote Provisioning Server.
+
+``mesh models rpr scan-stop``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Stop any ongoing scanning on the Remote Provisioning Server.
+
+``mesh models rpr link-get``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Get the current link status of the Remote Provisioning Server.
+
+``mesh models rpr link-close``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Close any open link on the Remote Provisioning Server.
+
+``mesh models rpr provision-remote <UUID> <NetKeyIndex> <addr>``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Provision a Mesh Node using PB-Remote
+
+	* ``UUID``: UUID of the unprovisioned node.
+	* ``NetKeyIndex``: Network Key Index to give to the unprovisioned node.
+	* ``addr``: Address to assign to remote device. If addr is 0, the lowest available address will be chosen.
+
+``mesh models rpr reprovision-remote <addr> [<comp changed: false, true>]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+	Reprovision a Mesh Node using PB-Remote
```

Comment:
```suggestion
	Reprovision a mesh node using the PB-Remote provisioning bearer.
```

Reply:
Will not be better to say "the Remote Provisioning Bearer (PB-Remote)" at the end.

Reply:
If you'd like :)
As long as you write "the remote provisioning bearer" instead (no title casing, at least not in the spec).